resharper/resharper-unity/src/CSharp/Daemon/UsageChecking/UsageInspectionsSuppressor.cs

```diff
@@ -15,6 +19,21 @@ public class UsageInspectionsSuppressor : IUsageInspectionsSuppressor
     {
         private readonly ILogger myLogger;
 
+        private readonly List<IClrTypeName> implicitlyUsedInterfaces = new List<IClrTypeName>()
```

Comment:
I was debating whether to add this to KnownTypes however I couldn't think of a case where you would need the same types. Any opinions?